unknown property "HADOOP_HOME" in "plugins\repository-hdfs\build.gradle" (Windows OS specific)

<!-- Bug report -->

**Plugins installed**: [default]

**JVM version** : `1.8.0_102`

**OS version** : Windows 7

**Description of the problem including expected versus actual behavior**: 

Issue is Windows specific. Gradle tasks throws out error "unknown Property "HADOOP_HOME" if HADOOP_HOME is set in the system.

Issue is due to referencing unknown property HADOOP_HOME in the exception statement

**Steps to reproduce**:

 1. Set HADOOP_HOME
 2. Make sure that any of the files winUtils.exe, hdfs.dll or hadoop.dll is not present in HADOOP_HOME\bin
 3. Run any of the gradle tasks
 4. Observe that you get error "unknown property HADOOP_HOME in plugins\repository-hdfs\build.gradle #line_number:40

**Provide logs (if relevant)**:

 FAILURE: Build failed with an exception.
* Where:
Build file 'C:\..\..\..\elasticsearch\plugins\repository-hdfs\build.gradle' line: 140

* What went wrong:
A problem occurred evaluating project ':plugins:repository-hdfs'.
> Could not get unknown property 'HADOOP_HOME' for project ':plugins:repository-hdfs' of type org.gradle.api.Project.

--- a/plugins/repository-hdfs/build.gradle
+++ b/plugins/repository-hdfs/build.gradle
@@ -137,7 +137,7 @@ if (Os.isFamily(Os.FAMILY_WINDOWS)) {
         Files.exists(path.resolve("bin").resolve("hdfs.dll"))) {
       fixtureSupported = true
     } else {
-      throw new IllegalStateException("HADOOP_HOME: " + path.toString() + " is invalid, does not contain hadoop native libraries in $HADOOP_HOME/bin");
+      throw new IllegalStateException("HADOOP_HOME: ${path} is invalid, does not contain hadoop native libraries in \$HADOOP_HOME/bin");
     }
   }
 } else {

